.circleci/config.yml

```diff
@@ -308,6 +308,13 @@ jobs:
             cd test/app/android/app
             echo $GOOGLE_SERVICES_JSON > google-services.json
             cd ../../../..
+      - run:
+          name: "Start relay server"
+          background: true
+          command: |
+            cd test/app
+            yarn run detox:relay-server
+            cd ../..
```

Comment:
Do we need to do this? I thought circleci defaults to working_directory with every run?

Reply:
The working directory would be the root of the project, not the nested `test/app` folder. I found that I needed this in the past, but things might have changed. The rest of the config file uses this pattern.

Reply:
Using `cd ../..` will take you back to the root, but I think the default is to start at root with each run so no need to go back. But yeah there is no harm in leaving this.